docs/03_tutorials/tic-tac-toe-game-contract.md

```diff
@@ -266,11 +255,11 @@ Let's open tic.tac.toe.cpp and set up the boilerplate:
 #include "tic.tac.toe.hpp"
 ```
 
-### Action Handler
+### Action Handlers
 
 We want `tic tac toe` contract to only react to actions sent to the `tic.tac.toe` account and react differently according to the type of the action. The actions that we declared previously are ***create***, ***move***, ***restart***, and ***close***. Let's define the individual action handlers in the next section.
 
-### "create" Action Handler
+### `create` Action Handler
```

Comment:
using ticks "`" is going to make the link to this title (in the page map on the right side of the document) render incorrectly
we have a general rule to not use ticks "`" in titles

Reply:
this sounds like a software issue. The tick is correct Markdown syntax

Reply:
yes the tick is correct from markdown point of view, but our doc generator is not handling it correctly right now, `it is a known issue` and as of now what we can do about it is to avoid using ticks in titles.

Reply:
Generally in headings (H1, H2, H3) and titles that go in primary or secondary navigation, we avoid double quotes or any emphasis syntax like back ticks and keep the heading words plain. If we use any form of emphasis in headings, for example, `create` Action Handler and if we have a software issue, the rendering will look awkward in the TOC and navigation. 

If we use double quotes in a H3, it will look like this in the navigation: 

### "create" Action Handler
It looks awkward to have any form of emphasis syntax in headings and titles. 

(Also, does it affect SEO search if headings and titles are with emphasis syntax? If somebody wants to search create action handler, and if the create is with "Create" or `create`, will it affect the search? I am not sure). 

So to elimate these problems, how about we use something plain like:

### Action Handler - Create
### Action Handler - Move
### Action Handler - Restart

Reply:
These actions are function name. They are not typical titles. I still think it is better to keep the action lower case

Reply:
this software issue should be to investigate though. I don't remember this behaviour happening on readme.io

Reply:
Yes, I agree to keep the function names in lowercase. The updated file looks good (Action Handler - create).